recipes/matrix_client/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "matrix_client" %}
+{% set version = "0.3.2" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: dce3ccb8665df0d519f08e07a16e6d3f9fab3a947df4b7a7c4bb26573d68f2d5
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - pytest-runner >=5.1
+    - python
+    - requests
+  run:
+    - python
+    - requests
+
+test:
+  imports:
+    - matrix_client
+  requires:
```

Comment:
These dependencies aren't being used. You should either add a commands section to run `pytest` or remove `requires`.

Reply:
Makes sense, removed them now.